datahub-ranger-plugin/README.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Configuring Apache Ranger"
+hide_title: true
+---
+# Configuring Apache Ranger
+Datahub Apache Ranger Plugin configuration is consist of below configuration sections 
+1. Configuration at Apache Ranger Deployment 
+2. Configuration at Datahub Deployment
+
+The configuration sections are tested on [Privacera Platform](https://privacera.com/) v6.3.0.1.
+
+# Assumption 
+- Apache Ranger and DataHub are configured with same IDP
+- Apache Ranger service is running on HTTP
+
+# Configuration 
+## Configuration at Apache Ranger Deployment 
+1. Download the datahub-ranger-plugin from [Maven](https://mvnrepository.com/artifact/io.acryl/datahub-ranger-plugin)
+2. Create  *datahub*  directory in each *Privacera Ranger* container 
```

Comment:
How about something like this:

Create a "datahub" directory inside the "ranger-plugins" directory where Apache Ranger is deployed. 

For example, to do this in a Privacera container, 
.....

Reply:
Done